requirements.txt

```diff
@@ -1,2 +1,4 @@
+boto3==1.26.41
+pymongo==4.3.3
```

Comment:
We don't usually pin this hard in the `requirements*.txt` files unless there is a real need to do so, in which case there would also need to be a comment explaining the reason.

Reply:
Got it. I was trying to avoid https://github.com/cisagov/findcdn/issues/54 which bit me in the ass recently!
I'll change it to >= current minor revision.

Reply:
Addressed in commit f006e82.